paymentsheet/src/test/java/com/stripe/android/paymentsheet/cvcrecollection/BacsMandateConfirmationContractTest.kt

```diff
@@ -0,0 +1,96 @@
+package com.stripe.android.paymentsheet.cvcrecollection
+
+import android.content.Intent
+import android.os.Build
+import androidx.core.os.BundleCompat
+import androidx.test.core.app.ApplicationProvider
+import com.google.common.truth.Truth.assertThat
+import com.stripe.android.paymentsheet.PaymentSheet
+import com.stripe.android.paymentsheet.paymentdatacollection.bacs.BacsMandateConfirmationContract
+import com.stripe.android.paymentsheet.paymentdatacollection.bacs.BacsMandateConfirmationContract.Args
+import com.stripe.android.paymentsheet.paymentdatacollection.bacs.BacsMandateConfirmationResult
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.mockito.MockitoAnnotations
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [Build.VERSION_CODES.TIRAMISU])
+class BacsMandateConfirmationContractTest {
+
+    private lateinit var contract: BacsMandateConfirmationContract
+
+    @Before
```

Comment:
I think you can remove this whole before block, remove the lateinit, and just assign it.